maskfile.md

```diff
@@ -78,7 +103,31 @@ fi
 echo "Tests passed!"
 ~~~
 
+~~~powershell
+param (
+    $file = $env:file,
+    $f = $env:f
+)
```

Comment:
Question: why are we checking for `$env:file` and `$env:f`? Only the option's name (`file`) is set as an env variable, not the short form. I noticed this in the other powershell blocks too, below and above.

Reply:
You are right, the parameter should probably just be `$f` rather then `$f = $f:env`.  Let me go ahead and verify this before I change it.

Reply:
Hmmm i'm still confused. What does `$f` do here?

I thought that you don't need `$f`/`$env:f` and therefore you wouldn't need this either:

```powershell
if ($f) {
    $file = $f
}
```

It looks like a temp variable or alias to me, but i'm not sure why it's needed 🤔 

Same question for `$w` and 

```powershell
if ($w) {
    $watch = $w
}
```

Reply:
I was just covering for the cases where there are shorthanded flags `-f` and `-w`. I guess maybe these are irrelevant since all of the incoming vars are being passed in through the environment.